pkg/controller/change/device/controller.go

```diff
@@ -158,17 +153,34 @@ func (r *Reconciler) reconcileRollback(change *devicechangetypes.DeviceChange) (
 
 // doRollback rolls back a change on the device
 func (r *Reconciler) doRollback(change *devicechangetypes.DeviceChange) error {
-	log.Infof("doRollback %v", change)
+	log.Infof("Execucting Rollback for %v", change)
 	deltaChange, err := r.computeNewRollback(change)
 	if err != nil {
 		return err
 	}
-	log.Infof("Rolling back %s with %s", change.Change, deltaChange)
-	setRequest, err := values.NativeNewChangeToGnmiChange(deltaChange)
+	log.Infof("Rolling back %v with %v", change.Change, deltaChange)
+	return r.translateAndSendChange(deltaChange)
+}
+
+func (r *Reconciler) translateAndSendChange(change *devicechangetypes.Change) error {
+	setRequest, err := values.NativeNewChangeToGnmiChange(change)
+	if err != nil {
+		return err
+	}
+	log.Info("Reconciler set request ", setRequest)
+	log.Info("Device ", change.DeviceID)
+	deviceTarget, err := southbound.GetTarget(devicetopo.ID(change.DeviceID))
+	if err != nil {
+		//TODO revisit change state.
+		log.Infof("Device %s is not connected, accepting change", change.DeviceID)
```

Comment:
I would suggest we still return the error here. The `DeviceChange` will be `FAILED` with an `ERROR`, but since we  currently treat all errors as retryable regardless of their context, the change will be retried by the network controller.